api/config/rolepermission.php

```diff
@@ -0,0 +1,532 @@
+                'en' => 'Super Admin',
+                'fr' => 'Super Admin'
+            ],
+            'description' => [
+                'en' => 'Makes teams, assigns roles to other users (including assigning users to orgs), manages lists of business data, and has the extraordinary ability to edit or delete other users.',
+                'fr' => 'Il crée des équipes, attribue des rôles à d\'autres utilisateurs (y compris l\'attribution d\'utilisateurs à des organisations), gère des listes de données commerciales et a la capacité extraordinaire de modifier ou de supprimer d\'autres utilisateurs.'
+            ]
+        ],
+    ],
+
+    /*
+    |--------------------------------------------------------------------------
+    | Seeders
+    |--------------------------------------------------------------------------
+    |
+    | A map of the values to seed with the shape:
+    |
+    |   'role_name' => [
+    |       'resource' => [
+    |           'scope' => [ 'actions' ]
+    |       ]
+    |   ]
+    |
+    */
+    'seeders' => [
+        'guest' => [
+            'classification' => [
+                'any' => ['view']
+            ],
+            'department' => [
+                'any' => ['view']
+            ],
+            'skill' => [
+                'any' => ['view']
+            ],
+            'skillFamily' => [
+                'any' => ['view']
+            ],
+            'publishedPoolAdvertisement' => [
+                'any' => ['view']
+            ],
+            'applicantCount' => [
+                'any' => ['view']
+            ],
+            'searchRequest' => [
+                'any' => ['create']
+            ],
+            'team' => [
+                'any' => ['view']
+            ],
+        ],
+
+        'base_user' => [
+            'classification' => [
+                'any' => ['view']
+            ],
+            'department' => [
+                'any' => ['view']
+            ],
+            'skill' => [
+                'any' => ['view']
+            ],
+            'skillFamily' => [
+                'any' => ['view']
+            ],
+            'user' => [
+                'own' => ['view', 'update']
+            ],
+            'publishedPoolAdvertisement' => [
+                'any' => ['view']
+            ],
+            'applicantCount' => [
+                'any' => ['view']
+            ],
+            'searchRequest' => [
+                'any' => ['create']
+            ],
+            'team' => [
+                'any' => ['view']
+            ],
+        ],
+
+        'applicant' => [
+            'application' => [
+                'own' => ['view', 'submit']
+            ],
+            'draftApplication' => [
+                'own' => ['create', 'delete']
+            ],
+            'submittedApplication' => [
+                'own' => ['archive']
+            ]
+        ],
+
+        'team_admin' => [
+            'user' => [
+                'any' => ['view']
```

Comment:
I don't think team_admin should have any `any-user` permissions.

Reply:
Oh, I think this is supposed to be userBasicInfo, not user

Reply:
That makes sense, updated in [fix team admin permissions](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5560/commits/f1cec7c1c76cef7ab4a0f9361e3f9e3664217582)